test/components/views/location/MapError-test.tsx

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
```

Comment:
What's the reason for switching testing libraries? From my perspective it's easier to work on unit tests if all suites are using the same frameworks, and in this case Enzyme is far more common

Reply:
We want to move toward rtl now, as Enzyme is [less maintained now ](https://github.com/enzymejs/enzyme/issues/2429#issuecomment-858010582).

I'll add a point to the web weekly agenda